e2e/cypress/tests/integration/channel/new_channel_with_board_spec.js

```diff
@@ -38,7 +38,7 @@ describe('Channel routing', () => {
         });
     });
 
-    it('MM-T5141 Channel URL validation for spaces between characters', () => {
+    it('MM-T5141 New Channel is created with an associated Board', () => {
```

Comment:
not necessary for this change, but the description I used when initially adding this test was not the most accurate, actually was the one I copied from a test I used as a template